.github/actions/shared-node-cache/action.yml

```diff
@@ -0,0 +1,33 @@
+name: 'Shared Node Cache'
+description: 'Install & cache our npm dependencies'
+
+# The inputs this action expects.
+inputs:
+  node-version:
+    description: 'Node version to use'
+    required: false
+    default: '20.x'
+  cache-bust:
```

Comment:
This appears unused... or does a value passed to it do enough to bust the cache?